packages/mjml-text/README.md

```diff
@@ -19,32 +30,36 @@ This tag allows you to display text in your email.
- padding                      | px            | supports up to 4 parameters                 | 10px 25px
- padding-top                  | px            | top offset                                  | n/a
- padding-bottom               | px            | bottom offset                               | n/a
- padding-left                 | px            | left offset                                 | n/a
- padding-right                | px            | right offset                                | n/a
- css-class                    | string        | class name, added to the root HTML element created | n/a
+ attribute                    | unit                    | description                                 | default value
+------------------------------|-------------------------|---------------------------------------------|-------------------------------------
+align      | in ["center", "justify", "left", "right"]      | horizontal text alignment <!--- not CSS "align" because different shape -->      | left
+color      | in [#RRGGBB, &lt;named-color&gt, rgb(), rgba()] <!--- see color.js in mjml-core -->      | text color; <!--- not CSS "color" because different shape -->  | #000000
+container-background-color    | in [#RRGGBB, &lt;named-color&gt, rgb(), rgba()] <!--- see color.js in mjml-core -->      | "mj-text" background color <!--- not CSS "color" because different shape -->      | n/a
+css-class      | string       | creates HTML classes and attributes, added to the root HTML element created; see "css-class" in [mj-style](#../mjml-head-style/mjml-style) for code example      | n/a
+font-family                   | string; font or generic family name(s)      | same as CSS "font-family"       | Ubuntu, Helvetica, Arial, sans-serif
+font-size      | px: not negative      | text size <!--- not CSS "font-size" because different shape -->      | 13px
+font-style        | often "normal", "italic", or "oblique &lt;angle&gt;"      | same as CSS "font-style"      | n/a
+font-weight                   | string            | font weight (or boldness); same as CSS "font-weight"      | n/a
+height      | px, %      | height of the "mj-text" element; % is relative to the containing block's height. <!--- not CSS "height" because different shape -->      | n/a
+letter-spacing       | px: pos, zero, or neg      | letter spacing within the line <!--- not CSS "letter-spacing" because different shape -->      | n/a
+line-height          | px, %       | space between lines; % is relative to the block's font <!--- not CSS "line-height" because different shape -->      | 100%
+padding              | px, % {1,4}      | sets "padding-bottom", "-left", "-right", and "-top"; % is relative to "width" of containing block <!--- not CSS "padding" because different shape -->      | 10px 25px
+padding-bottom       | px, %                            | sets bottom padding; % same as above        | 10px
```

Comment:
Padding are not supported in % in some desktop client, I think they are described as px only in the code ?

Reply:
Should we match `packages/mjml-text/src/index.js > allowedAttributes`? Lines 20-24, part of `allowedAttributes` are

>'padding-bottom': 'unit(px,%)',
    'padding-left': 'unit(px,%)',
    'padding-right': 'unit(px,%)',
    'padding-top': 'unit(px,%)',
    padding: 'unit(px,%){1,4}',

Reply:
ok my bad looks like we've put px  & padding on everything let's keep it that way